### ISSUE TITLE
[Meds / Accessibility] Address missing alt text for medication image on both the Meds List and Meds Details pages

### ISSUE THREAD

**Issue by iseabock** [department-of-veterans-affairs/vets-website#40330]
## Context
During Secure Messaging Prescription Renewal Flow a non-launch blocking issue was noted that the alt text for the medication image is not there. 

## Background
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/125855

## Description 
Address missing alt text for medication image on both the Meds List and Meds Details pages

## To-Do's
- [ ] Check with Sarah Horton to see if there's additional context we should provide when adding in alt text

## Acceptance Criteria
- [ ] Determine if the image has the appropriate meta data to include in the alt text
- [ ] If it does add it
- [ ] Otherwise, report the details of the investigation 
- [ ] Accessibility review

**Comment by ckeif-iat** [comment]
I have been investigating this, but unfortunately it does not seem possible to reflect the full packaging text shown in the image in the alt text as described in [#125855](https://github.com/department-of-veterans-affairs/va.gov-team/issues/125855) with the data we have available today.

I've opened [this thread](https://dsva.slack.com/archives/C04PRFEJQTY/p1768323257378849) with my findings so far and some suggestions on what we could do today.
